index.html

```diff
@@ -40,9 +44,179 @@ <h1 class="description">I am Desmond, glad to see you!
           <li><a href="#"><img src=" ./images/M-icon.svg" alt="M-icon"></a> </li>
           <li><a href="#"><img src=" ./images/hand-raise.svg" alt="handraise icon"></a></li>
         </ul>
-      </section>
-
-      
+      </section> 
     </main>
+    <!-- work section -->
+    <section class="work">
+        <div class="container">
+            <div class="workSection">
+                <!-- Card 1 section -->
+                <div class="card ">
+                    <img class="card-image" src="./images/skills-snapshot.png" alt="skills-snapshot">
+                    <div class="card-content">
+                        <h1 class="card-title">Tonic</h1>
+                        <div class="skills">
+                            <span>CANOPY</span>
+                            <span class="dot"></span>
+                            <span>Back End Dev</span>
+                            <span class="dot"></span>
+                            <span>2015</span>
+                        </div>
+                        <div class="card-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </div>
```

Comment:
- [ ] It is a good idea to use HTML semantic tag on the page whenever possible, as this will improve the SEO of your page. Kindly replace the `<div>` here with a `<p>` tag.